docs/3.0/runtime.md

```diff
@@ -0,0 +1,80 @@
+---
+---
```

Comment:
Dear lord is this on old issue: https://github.com/eclipsesource/tabris-js/issues/391

I'm not sure if this would be useful to integrate or not: https://github.com/eclipsesource/tabris-js/files/1010658/tabris2.svg.txt